packages/collector/test/tracing/cloud/aws/sqs/test.js

```diff
@@ -36,7 +36,7 @@ mochaSuiteFn('tracing/cloud/aws/sqs', function() {
   globalAgent.setUpCleanUpHooks();
   const agentControls = globalAgent.instance;
 
-  describe('tracing enabled, no suppression', function() {
+  describe.only('tracing enabled, no suppression', function() {
```

Comment:
I use `.only` a lot locally, but usually try to not commit it. `eslint` will fail due to it. (There is rule that forbids only so it is not accidentally committed)